packages/superset-ui-core/package.json

```diff
@@ -55,6 +55,7 @@
     "lodash": "^4.17.11",
     "pretty-ms": "^7.0.0",
     "react-error-boundary": "^1.2.5",
+    "react-markdown": "^4.3.1",
```

Comment:
Could you update `yarn.lock` as well?

Reply:
yarn.lock is git ignored for me. or do you mean the one in the root directory?

Reply:
ah, i see the yarn.lock in root is updated a lot. i'll do that